src/contracts/README.md

```diff
@@ -0,0 +1,13 @@
+## Contracts for RIP-7560 and ERC-4337
+
+### Compile
+
+```shell
+$ forge compile
+```
+
+### Deploy
+
+```shell
+$ forge script script/Deploy.s.sol:CounterScript --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast
+```
```

Comment:
And since there is no default loading part of env like bun, why don't you explicitly put this part in the script?

source .env && forge script script/Deploy.s.sol:DeployScript --rpc-url $RPC_URL --private-key $PRIVATE_KEY --broadcast